maplibre/src/coords.rs

```diff
@@ -139,7 +153,9 @@ impl TileCoords {
     /// The [`TileCoords`] `T(x=5,y=5,z=0)` exceeds its bounds because there is no tile
     /// `x=5,y=5` at zoom level `z=0`.
     pub fn into_world_tile(self, scheme: TileAddressingScheme) -> Option<WorldTileCoords> {
-        let bounds = 2i32.pow(self.z as u32);
+        // FIXME: MAX_ZOOM is 32, which means max bound is 2^32, which wouldn't fit in u32 or i32
+        // Note that unlike WorldTileCoords, values are signed (no idea why)
```

Comment:
I think we could make WorldTileCoords unsigned.

Reply:
I got a bit confused by why you have two near-identical implementations here tbh.  Sure, an unsigned would work just fine.

Reply:
lets discuss here: https://github.com/maplibre/maplibre-rs/issues/65